ASF-STM.user.js

```diff
@@ -1154,5 +1215,6 @@
         let anchor = document.getElementsByClassName("profile_small_header_texture")[0];
         anchor.appendChild(buttonDiv);
         enableButton();
+        GM_addStyle(css);
```

Comment:
I would like this script to be compatible with Greasemonkey too, and it does not have an addStyle api. Could you please use pure js solution instead? Like using https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet/insertRule or just by adding an inline style to the filter panel?